packages/service-library/src/servicelib/aiohttp/application_setup.py

```diff
@@ -89,7 +89,7 @@ def _is_addon_enabled_from_config(
         for part in parts:
             if section and part == "enabled":
                 # if section exists, no need to explicitly enable it
-                return strtobool(f"{searched_config.get(part, True)}")
+                return parse_obj_as(bool, f"{searched_config.get(part, True)}")
```

Comment:
There is no need to convert to ``str`` here, it suffices with
```python
parse_obj_as(bool,  searched_config.get(part, True))
```

Reply:
true that.